deferrable_gratification.gemspec

```diff
@@ -24,11 +24,11 @@ Features include:
 
   gem.license = 'MIT'
 
-  gem.required_ruby_version = '>= 1.8.7'
+  gem.required_ruby_version = '>= 3.0.6'
 
   gem.add_dependency 'eventmachine', '>= 1.0.3'
 
-  gem.add_development_dependency "bundler", "~> 1.5"
+  gem.add_development_dependency "bundler", "~> 2.2.33"
```

Comment:
It might be easier to just drop off the bundler and rspec version restrictions below 🤔